src/normalinversegamma.jl

```diff
@@ -43,7 +43,7 @@ end
 
 function mode(d::NormalInverseGamma)
     mu = d.mu
-    sig2 = d.scale / (d.shape + 1.0)
+    sig2 = d.scale / (d.shape - 0.5)
```

Comment:
This can became negative for small shape??

Reply:
I don't think that can be correct by looking at `pdf(D, mode(D)... +- eps())`

Reply:
hm, good point. maybe the mode is at 0 in that case?

Reply:
...or rather infinite...

Reply:
...which makes no sense now that I've thought about it for more than 5 seconds